web_view_calendar_list/static/src/js/calendar_list_controller.js

```diff
@@ -0,0 +1,11 @@
+odoo.define('web_view_calendar_list.CalendarListController', function (require) {
```

Comment:
Is this file necessary?

Reply:
I made it just in case

Reply:
Also, I think it is a good approach to define an specific class, but it is not used right now.